report_merged_qc_results.py

```diff
@@ -71,49 +103,32 @@
     ("TMREDS", "Sickle Cell Disease REDS III"),
 ]
 
-# columns in weekly report tab3 'Production Metrics'
-# External ID  # extract from merge_name
-# Collection  # Study/Cohort
-# PF HQ Aligned Q20 Bases # new
-# Mean Insert Size (Library AVG)  # new
-# Mean Coverage (Raw)  # new correct value
-# WGS HET SNP Q  # new
-# WGS HET SNP SENSITIVITY  # new
-# Per 10 Coverage Bases
-# Per 20 Coverage Bases
-# Q20 Bases
-# Contamination %  # convert Contamination Rate to Contamination %
-# Notes
-
 
 def main():
     args = parse_args()
-    run(args.recent_merge_report, args.new_metrics_file, args.output_file)
+    run(args.recent_merge_report, args.output_file)
 
 
 def parse_args():
-    parser = argparse.ArgumentParser(description=__doc__)
-    parser.add_argument(
-        "recent_merge_report", help="*.xlsx, usually generated by QC group"
+    parser = argparse.ArgumentParser(
+        description=__doc__,
+        formatter_class=argparse.RawDescriptionHelpFormatter
     )
     parser.add_argument(
-        "new_metrics_file", help="*.xlsx, usually generated by R&D group"
+        "recent_merge_report",
+        help="*.xlsx, usually downloaded from Exemplar LIMS",
     )
     parser.add_argument("output_file", help="should end with .xlsx")
```

Comment:
I got `NameError` message when I changed to:
```
--recent_merge_report
--output_file
```

Reply:
That's weird. I don't see how that would cause an error.
Is there a traceback?
The change shouldn't be too much:
```
parser.add_argument(
        "new_metrics_file", help="*.xlsx, usually generated by R&D group"
        "--recent_merge_report",
        help="*.xlsx, usually downloaded from Exemplar LIMS",
    )
```

Reply:
I thought too.  Actually arguments error.
```
usage: report_merged_qc_results.py [-h]
                                   [--recent_merge_report RECENT_MERGE_REPORT]
                                   [--output_file OUTPUT_FILE]
report_merged_qc_results.py: error: unrecognized arguments: tmreds_merge_report.xlsx r4.xlsx
```

Reply:
Oh you would need to do to run it:
`python report_merged_qc_results.py --recent_merge_report REPORT_GOES_HERE --output_file OUTPUT_FILE`

Reply:
Got it.  New commit has been pushed.